python/ray/tests/test_concurrency_group.py

```diff
@@ -179,6 +180,28 @@ def ping(self):
     print(ray.get(n.ping.remote()))
 
 
+def test_invalid_concurrency_group():
```

Comment:
add a note for why you are using the `run_string_as_driver` test path here

Reply:
updated